tests/buffer_ut/buffer_ut.c

```diff
@@ -1561,6 +1558,87 @@ BEGIN_TEST_SUITE(Buffer_UnitTests)
         BUFFER_delete(res);
     }
 
+    // Codes_SRS_BUFFER_07_029: [ BUFFER_create_with_size shall create a BUFFER_HANDLE with a pre allocated underlying buffer size.]
```

Comment:
>Codes [](start = 7, length = 5)

Tests

Reply:
Here and everywhere.

---
In reply to: [239323121](https://github.com/Azure/azure-c-shared-utility/pull/238#discussion_r239323121) [](ancestors = 239323121)